pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -186,6 +193,28 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	return ResultNormal, nil
 }
 
+func updateLabels(subnetSet *v1alpha1.SubnetSet, systemNs bool) bool {
+	metadataChanged := false
+	if subnetSet.Labels == nil {
+		return metadataChanged
+	}
+	if value, exists := subnetSet.Labels[servicecommon.LabelDefaultSubnetSet]; exists {
+		log.Debug("Adding default-network label", "SubnetSet", subnetSet.Name)
+		// reserve the old label for other controllers compatibility
```

Comment:
Maybe add TODO comments for using systemNs to determine if the old label will be kept after other controllers compatibility are resolved?